external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -262,7 +292,33 @@ export type TraceEvent =
         bcodeLoc?: ILoc,
         kind: TraceInstructionKind
     }
-    | { type: TraceEventKind.Effect, effect: EventEffect };
+    | { type: TraceEventKind.Effect, effect: EventEffect }
+    | { type: TraceEventKind.PTBStart, id: number, summary: PTBSummary[] }
+    | { type: TraceEventKind.PTBCommand, command: PTBCommandInfo };
+
+export type PTBCommandInfo =
+    | {
+        kind: PTBCommandKind.MoveCallStart
+    } | {
+        kind: PTBCommandKind.MoveCallEnd
+    } | {
+        kind: PTBCommandKind.SplitCoinsStart
```

Comment:
We may consolidate info fro SplitCoins and Transfer but since the trace format for external events is still a bit in flux, let's wait and see